feature-libs/checkout/base/components/checkout-delivery-address/checkout-delivery-address.component.ts

```diff
@@ -89,6 +89,20 @@ export class CheckoutDeliveryAddressComponent implements OnInit {
       region = address.region.isocode + ', ';
     }
 
+    let numbers = '';
+    if (address.cellphone && address.phone) {
+      numbers =
+      `P: ${address.phone}
+      M: ${address.cellphone}`;
+      if(address.cellphone === address.phone) {
+        numbers = 'M: ' + address.cellphone;
+      }
+    } else if (address.cellphone && !address.phone) {
+      numbers = 'M: ' + address.cellphone;
+    } else if (!address.cellphone && address.phone) {
```

Comment:
Because of line 93 and line 100, we can truncate this to:

```suggestion
    } else if (address.phone) {
```